assets/js/base/components/radio-control-accordion/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { withInstanceId } from '@woocommerce/base-hocs/with-instance-id';
+
+/**
+ * Internal dependencies
+ */
+import RadioControlOption from '../radio-control/option';
+import '../radio-control/style.scss';
```

Comment:
I'm curious why this style is imported here. Since we're building to the same bundle, the import in `radio-control/style.scss` should be sufficient. So I don't think you'll need this import.

Reply:
TIL. OK, I need to put webpack and bundling as the next item on my learning list.